library/common-library/DEVELOPERS.md

```diff
@@ -99,6 +99,37 @@ spec:
 
 
 
+## _annotations.tpl
+### `newrelic.common.annotations.deployment` and `newrelic.common.annotations.pod`
+These are functions that are used to annotate objects. They are configured by this `values.yaml`
+```yaml
+global:
+  deploymentAnnotations: {}  # included in all the deployments and daemonsets of all the charts
+  podAnnotations: {}  # included in all the pods of all the charts
+deploymentAnnotations: {}  # included in all the deployments and daemonsets of the chart that the chart writes is implementing
```

Comment:
the comment is a bit confusing

Reply:
"of the chart that the chart writes is implementing"?

Reply:
> of the chart that chart writer is implementing

I copied and pasted this comment from the labels so there could be an issue in that part too. I am rechecking.